ipatests/prci_definitions/nightly_ipa-4-10_previous.yaml

```diff
@@ -1774,6 +1774,19 @@ jobs:
         timeout: 5000
         topology: *master_2repl_1client
 
+  fedora-previous-ipa-4-10/test_sso:
+    requires: [fedora-previous-ipa-4-10/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-previous-ipa-4-10/build_url}'
+        update_packages: True
```

Comment:
lint detected an issue:
```
[File] nightly_ipa-4-10_previous [Jobs] 148
ERROR: In job 'fedora-previous-ipa-4-10/test_sso':
  'update_packages' field should be set to false or not defined
```

Reply:
I missed that last night.  Fixed this issue earlier today.